nl/seg6_linux.go

```diff
@@ -0,0 +1,121 @@
+package nl
+
+import (
+	"errors"
+	"fmt"
+	"net"
+)
+
+type Ipv6SrHdr struct {
+	nexthdr       uint8
```

Comment:
This project in most parts seems to be following the convention of CamelCase naming.
Please avoid underscore in structure field names and change this one to something like `hdrLen` or `headerLen` what you prefer.

Reply:
Changed to below.
```
type IPv6SrHdr struct {
    nextHdr       uint8
    hdrLen        uint8
    routingType   uint8
    segmentsLeft  uint8
    firstSegment  uint8
```